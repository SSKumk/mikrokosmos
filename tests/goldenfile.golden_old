[1;34mLoading tests/testing.mkr...[m
color mode: off[m
Î»a.a â‡’ I, ifelse, id[m[m
Î»a.a â‡’ I, ifelse, id[m[m
Î»a.Î»b.b â‡’ nil, 0, false[m[m
Î»a.Î»b.b â‡’ nil, 0, false[m[m
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6[m[m
Î»a.Î»b.Î»c.Î»d.Î»e.(((a (a (a b))) c) (((a (a b)) ((a (a (a b))) c)) d)) e[m
Î»a.Î»b.a â‡’ K, true, const[m[m
Î»a.a â‡’ I, ifelse, id[m[m
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6[m[m
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6[m[m
types: on[m
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6[m :: (A â†’ A) â†’ A â†’ A[m[m
verbose: on[m
(plus 3) 2
((Î»Î»Î»Î»((4 2) ((3 2) 1)) Î»Î»(2 (2 (2 1)))) Î»Î»(2 (2 1)))
(Î»Î»Î»((Î»Î»(2 (2 (2 1))) 2) ((3 2) 1)) Î»Î»(2 (2 1)))
Î»Î»((Î»Î»(2 (2 (2 1))) 2) ((Î»Î»(2 (2 1)) 2) 1))
Î»Î»(Î»(3 (3 (3 1))) (Î»(3 (3 1)) 1))
Î»Î»(2 (2 (2 (Î»(3 (3 1)) 1))))
Î»Î»(2 (2 (2 (2 (2 1)))))

Î»a.Î»b.a (a (a (a (a b)))) â‡’ 5[m :: (A â†’ A) â†’ A â†’ A[m[m
