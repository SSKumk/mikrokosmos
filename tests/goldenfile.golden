[1;34mLoading tests/testing.mkr...[m
color mode: off
Î»a.a â‡’ I, ifelse, id
Î»a.a â‡’ I, ifelse, id
Î»a.Î»b.b â‡’ nil, 0, false
Î»a.Î»b.b â‡’ nil, 0, false
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6
Î»a.Î»b.Î»c.Î»d.Î»e.(((a (a (a b))) c) (((a (a b)) ((a (a (a b))) c)) d)) e
Î»a.Î»b.a â‡’ K, true, const
Î»a.a â‡’ I, ifelse, id
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6
types: on
Î»a.Î»b.a (a (a (a (a (a b))))) â‡’ 6 :: (A â†’ A) â†’ A â†’ A
verbose: on
(plus 3) 2
((Î»Î»Î»Î»((4 2) ((3 2) 1)) Î»Î»(2 (2 (2 1)))) Î»Î»(2 (2 1)))
(Î»Î»Î»((Î»Î»(2 (2 (2 1))) 2) ((3 2) 1)) Î»Î»(2 (2 1)))
Î»Î»((Î»Î»(2 (2 (2 1))) 2) ((Î»Î»(2 (2 1)) 2) 1))
Î»Î»(Î»(3 (3 (3 1))) (Î»(3 (3 1)) 1))
Î»Î»(2 (2 (2 (Î»(3 (3 1)) 1))))
Î»Î»(2 (2 (2 (2 (2 1)))))

Î»a.Î»b.a (a (a (a (a b)))) â‡’ 5 :: (A â†’ A) â†’ A â†’ A
[m